### PR TITLE
Add include field to package-references

### PR DESCRIPTION
**Pull request by gayaldassanayake**
## Purpose
$subject

### CODE REVIEW

File: learn/guides/organize-ballerina-code/package-references.md [hunk 1]
```diff
@@ -105,6 +112,12 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
 
 *   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.
 
+### The `include` field
+
+You can provide paths to any additional resources that needs to be packed in the BALA file during the use of `bal pack` command.
```
Comment (praneesha) [inline learn/guides/organize-ballerina-code/package-references.md:117]:
```suggestion
You can provide paths to any additional resources, which need to be packed in the BALA file during the use of the `bal pack` command.
```

File: learn/guides/organize-ballerina-code/package-references.md [hunk 2]
```diff
@@ -105,6 +112,12 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
...
+
+The include field is an array that accepts relative paths from the root of the package to the files or directories.
```
Comment (praneesha) [inline learn/guides/organize-ballerina-code/package-references.md:119]:
```suggestion
The `include` field is an array, which accepts relative paths from the root of the package to the files or directories. 
```

File: learn/guides/organize-ballerina-code/package-references.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ redirect_from:
 ├── modules/
 │     ├── model/
 │     └── module1.test/
+├── documents/
```
Comment (azinneera) [inline learn/guides/organize-ballerina-code/package-references.md:35]: Shall we change this to have one directory and one file in the package root

File: learn/guides/organize-ballerina-code/package-references.md (hunk 2 above)
Comment (azinneera) [inline learn/guides/organize-ballerina-code/package-references.md:119]: The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.

File: learn/guides/organize-ballerina-code/package-references.md [hunk 4]
```diff
@@ -105,6 +112,12 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
 
 *   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.
 
+### The `include` field
+
+You can provide paths to any additional resources that needs to be packed in the BALA file during the use of `bal pack` command.
+
+The include field is an array that accepts relative paths from the root of the package to the files or directories. 
+The included files and directories will have a file structure similar to the orginal package from the root of the BALA file.
```
Comment (azinneera) [inline learn/guides/organize-ballerina-code/package-references.md:120]: The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory. The included file paths will be packaged into the root of the BALA preserving its original structure.

File: learn/guides/organize-ballerina-code/package-references.md (hunk 3 above)
Comment (gayaldassanayake) [reply learn/guides/organize-ballerina-code/package-references.md:35]: Updated to have,
├── documents/
├── sample.png

**Comment by gayaldassanayake** [comment]
@azinneera @praneesha added descriptions for `export` and `icon` as well. Please review.

File: learn/guides/organize-ballerina-code/package-references.md [hunk 5]
```diff
@@ -105,6 +110,25 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
 
 *   Once the package is production-ready, you can use a stable version (E.g. 1.0.0). Any subsequent minor or patch releases of the same major version should be backward compatible and, should not break existing builds.
 
+### The `export` field
+
+You can specify the modules which should be visible to the outside using the `export` field. The export field accepts a string array, which contains the names of the modules which
+need to be publicly visible.
+
+The modules should be referred in the format <pacakge_name>.<module_name>. The default module should be referenced by the package name.
```
Comment (praneesha) [inline learn/guides/organize-ballerina-code/package-references.md:118]:
```suggestion
The modules should be referred in the `<pacakge_name>.<module_name>` format. The default module should be referenced by the package name.
```

File: learn/guides/organize-ballerina-code/package-references.md [hunk 6]
```diff
@@ -105,6 +110,25 @@ Ballerina strictly follows the rules of [Semantic Versioning](https://semver.org
...
+
+### The `icon` field
+
+The `icon` field accepts a path to an icon. The specified icon will be packaged into the `docs/` directory of the BALA.
+
+Only the .png format is supported for the icon.
```
Comment (praneesha) [inline learn/guides/organize-ballerina-code/package-references.md:124]:
```suggestion
Only the `.png` format is supported for the icon.
```

**Review by praneesha — APPROVED** [review]

File: learn/guides/organize-ballerina-code/package-references.md [hunk 7]
```diff
@@ -45,6 +48,8 @@ org = "samjs"
 name = "winery"
 version = "0.1.0"
 export = ["winery", "winery.model"]
+include = ["documents", "sample.png"]
+icon = “icon.png”
```
Comment (azinneera) [inline learn/guides/organize-ballerina-code/package-references.md:52]: The symbol for `"` here seems to be wrong

**Pull request merged**